Google Play Music/CoreMusicApp.Hooks.cs

```diff
@@ -88,7 +88,12 @@ void gkh_KeyUp(object sender, KeyEventArgs e)
                         zoomInProgress = false;
                     }
                     break;
-
+                case Keys.F:
+                    if((ModifierKeys & Keys.Control) == Keys.Control)
```

Comment:
Please add the ` && ApplicationIsActivated()` from the above checks.  These keyboard hooks are global so in its current state Ctrl + F will go to the search box no matter what application you are currently in.